Realm.PCL/Schema/RealmSchemaPCL.cs

```diff
@@ -22,21 +22,43 @@
 
 namespace Realms
 {
+    /// <summary>
+    /// Describes the complete set of classes which may be stored in a Realm, either from assembly declarations or, dynamically, by evaluating a Realm from disk.
+    /// </summary>
+    /// <remarks>
+    /// By default this will be all the RealmObjects in all your assemblies unless you restrict with RealmConfiguration.ObjectClasses. 
+    /// Just because a given class <em>may</em> be stored in a Realm doesn't imply much overhead. There will be a small amount of metadata 
```

Comment:
Sure, I just mean that this is not a place where you would look for this information. The overhead concern is not limited to people who define specific schemas and you are unlikely to encounter this class if you don't (even then it's unlikely as you just need `RealmConfiguration`).